src/main/java/core/basesyntax/figures/FigureGenerator.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Random;
+
+public class FigureGenerator {
+    private static final int count = 5; //Number of inherited classes
```

Comment:
```suggestion
    private static final int FIGURES_AMOUNT / FIGURES_QUANTITY = 5; 
```
make code self description, constant naming - https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names